### ISSUE TITLE
Frontend: error handling with error pages

### ISSUE THREAD

**Issue by annikakrause** [amosproj/amos2022ss04-digital-identity#100]
## User story

1. As a {HR employee}
2. I want / need {error handling and error messages in case of occuring errors}
3. So that {I can recognize what went wrong and maybe prevent errors}

## Acceptance criteria

- create error pages for the http error codes (https://en.wikipedia.org/wiki/List_of_HTTP_status_codes#4xx_client_errors)
   - 400
   - 401
   - 403
   - 404
   - 408
   - 418
   - 500
   - 503
- these pages should be displayed at the path '/error/\<error-code\>'

## Definition of done (DoD)

- Added only after week 5
- The same for all features
- Here goes the project specific part

## DoD general criteria

- Feature has been fully implemented
- Feature has been merged into the mainline
- All acceptance criteria were met
- Product owner approved features
- All tests are passing
- Developers agreed to release

**Comment by annikakrause** [comment]
@TamaraHoock @veitmo this could be a good additional issue for sprint 6
